src/components/atoms/Page/Page.component.tsx

```diff
@@ -14,7 +14,13 @@ const Page = React.forwardRef<HTMLDivElement, PageProps>(({ ...props }, ref) =>
 
   return (
     <VisibilityComponent visibility={responsiveVisibility} isPreview={isPreview}>
-      <PageWrapper minHeight={minHeight} {...rest} ref={ref}>
+      <PageWrapper
+        className={isPreview ? 'preview-page' : undefined}
```

Comment:
why we need this ? what happend in real project

Reply:
@zrg-team In real project, drawer component will attach to body, while on preview it will attach to that page component.